packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -14,6 +14,7 @@
  */
 
 import {resolveDpr} from '@google/model-viewer/lib/utilities.js';
+import * as Filament from 'filament';
```

Comment:
We tend to avoid blanket imports like this because it can make tree-shaking/code-splitting less efficient. Would you be willing to change this to explicitly named imports instead (e.g., `import {Engine, Scene, Renderer, ...} from 'filament'`)?

Reply:
This was tricky but I got it to work!